src/Sylius/Bundle/CoreBundle/Resources/config/services/providers.xml

```diff
@@ -67,5 +67,11 @@
             <argument type="service" id="sylius.repository.order" />
             <tag name="sylius.statistics.orders_totals_provider" type="year" />
         </service>
+
+        <service id="sylius.provider.configurable_grid_provider" class="Sylius\Bundle\CoreBundle\Grid\Provider\ConfigurableProvider">
+            <argument type="tagged_locator" tag="sylius.grid_provider" index-by="key" />
```

Comment:
Does it tag itself and gets itself as argument or I misunderstand it?

Reply:
Yes, this is correct. We even register it with the same key as the default YAML provider but with a higher priority.

Reply:
Yes the idea is to be the first grid provider. So it need to be before those ones that are already configured in the grid package.